src/state/hooks.ts

```diff
@@ -498,3 +500,80 @@ export const useGetProposalLoadingStatus = () => {
   const votingStatus = useSelector((state: State) => state.voting.proposalLoadingStatus)
   return votingStatus
 }
+
+// Lottery
+export const useGetCurrentLotteryId = () => {
+  return useSelector((state: State) => state.lottery.currentLotteryId)
+}
+
+export const useGetUserLotteriesGraphData = () => {
+  return useSelector((state: State) => state.lottery.userLotteryData)
+}
+
+export const useGetUserLotteryGraphRoundById = (lotteryId: string) => {
+  const userLotteriesData = useSelector((state: State) => state.lottery.userLotteryData)
+  return userLotteriesData.rounds.find((userRound) => userRound.lotteryId === lotteryId)
+}
+
+export const useGetLotteriesGraphData = () => {
+  return useSelector((state: State) => state.lottery.lotteriesData)
+}
```

Comment:
Duplicate of line 509?

Reply:
509 is `userLotteryData`
this is `lotteriesData`

Reply:
Yea it's the two different subgraph data points - one is user-specific (`userLotteryData`), the other is 'all lotteries' (`lotteriesData`)